src/librustc_data_structures/steal.rs

```diff
@@ -20,6 +21,7 @@ use rustc_data_structures::sync::{RwLock, ReadGuard, MappedReadGuard};
 /// -- once the value is stolen -- it will never be read from again.
 //
 // FIXME(#41710): what is the best way to model linear queries?
+#[derive(Clone, Debug)]
```

Comment:
Why do we need to clone Steals now? It seems a bit weird to clone a Steal since what you probably actually want is something like `steal.borrow().clone()`, right? i.e., cloning the inside

Reply:
Probably we don't. I can check that.